app/youtube-grabber.js

```diff
@@ -49,11 +49,14 @@ class YoutubeGrabber {
     }
 
     let subscriberText
-
-    if (typeof (channelHeaderData.subscriberCountText.runs) !== 'undefined') {
-      subscriberText = channelHeaderData.subscriberCountText.runs[0].text
+    if (channelHeaderData.subscriberCountText && channelHeaderData.subscriberCountText) {
```

Comment:
This statement is checking the same condition twice.  Let's remove the excess condition if it isn't needed.